src/components/common/GallerySlider/GallerySlider.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import styles from './GallerySlider.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faChevronLeft,
+  faChevronRight,
+  faShoppingBasket,
+  faExchangeAlt,
+  faEye,
+  faStar,
+} from '@fortawesome/free-solid-svg-icons';
+import { faStar as farStar, faHeart } from '@fortawesome/free-regular-svg-icons';
+import Button from '../Button/Button';
+import { useSelector } from 'react-redux';
+import { getAll } from '../../../redux/productsRedux';
+import clsx from 'clsx';
+
+const GallerySlider = () => {
+  const topSeller = useSelector(getAll);
```

Comment:
getAll?

Reply:
Dobra niech tak zostanie skoro to tylko wygląd, a na podstawie tego taska to nawet jeszcze nie wiadomo jak to ma działać